subprojects/language-java/src/integTest/groovy/org/gradle/java/compile/incremental/AbstractJavaGroovyIncrementalCompilationSupport.groovy

```diff
@@ -24,9 +24,7 @@ abstract class AbstractJavaGroovyIncrementalCompilationSupport extends AbstractI
 
     def enableCompilationAvoidance() {
         if (language == CompiledLanguage.GROOVY) {
-            executer.beforeExecute {
-                executer.withArgument("-Dorg.gradle.groovy.compilation.avoidance=true")
-            }
+            settingsFile << "\nenableFeaturePreview('GROOVY_COMPILATION_AVOIDANCE')\n"
```

Comment:
This is an example where `FeaturePreviewsFixture` should be used instead.

Reply:
Thanks. Used `FeaturePreviewsFixture`.